dags/main_summary.py

```diff
@@ -642,5 +679,6 @@
 
 taar_locale_job.set_upstream(clients_daily_v6)
 taar_collaborative_recommender.set_upstream(clients_daily_v6)
+taar_ensemble.set_upstream(dag_weekly)
```

Comment:
The dag is already an argument to the operator.

Reply:
I've cleaned up the dag attribute assignment now in the constructor and set the upstream to be clients_daily.